### ISSUE TITLE
:bug: typeit integration not working

### ISSUE THREAD

**Issue by nunocoracao** [nunocoracao/blowfish#406]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Hugo & Blowfish versions**
Which version of Hugo and Blowfish are you using?

**Additional context**

**Comment by adir1** [comment]
Me again, apologies. You probably seen this, but apparently --disableFastRender actually makes things worse! I tracked down that .Page.HasShortcode doesn't work correctly with it. Furthermore, looks like auto-rebuild also breaks this HasShortcode function in many cases, so potentially my TypeIt issues were coming from that!
I restarted, cleaned and restarted many times, as you can image. 
TypeIt works properly whenever HasShortcode function properly works. However there is new error that stays consistently: "Uncaught ReferenceError: renderTypeItElements is not defined at HTMLScriptElement.onload"
To recap - it is minor because TypeIt still works properly regardless of this error. 
Thanks Again!

**Comment by nunocoracao** [comment]
No, there was a bug in the code that injects the import. The bug you mentioned just now is because I forgot to remove code from my experiments. Can you open a bug issue for it please @adir1?
